openmrs/src/test/java/org/motechproject/openmrs/it/MRSTasksIntegrationBundleIT.java

```diff
@@ -404,6 +442,40 @@ private TaskActionInformation prepareCreatePatientActionInformation(String adres
         return actionInformation;
     }
 
+    private TaskActionInformation prepareCreatePatientForEncounterActionInformation(String adress1, String motechId){
```

Comment:
use existing method for creating patient action information. Try to parameterize it